scripts/crdb_restore.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+BACKUP=$1
+if [[ $BACKUP == "" ]]; then
+  echo "No backup name given. It should look like '2020-01-29'."
+  exit 1
+fi
+
+# Set the environment:
+set -o allexport
+source ../.env
```

Comment:
Will that relative path work when running this from crontab?

You could use this
```bash
# get current working directory (pwd doesn't cut it)
cwd=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
```

https://github.com/NebulousLabs/skynet-webportal/blob/master/scripts/portal-upgrade.sh#L6

Reply:
Or maybe I can hard-code the path to the .env file? What would be better?

Reply:
Done, used your approach. It's less likely that we'll move the script (within the project) than that we'll deploy somewhere else.